### PR TITLE
test(heatmap-layer): add basic tests

### PR DESCRIPTION
**Pull request by viddity**
I grabbed a very small component to get started with contributing to this repo.
=> PR only adds 2 tests, no further changes.

### CODE REVIEW

File: __tests__/components/HeatmapLayer.test.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    expect(props.minZoomLevel).toStrictEqual(testProps.minZoomLevel);
+    expect(props.maxZoomLevel).toStrictEqual(testProps.maxZoomLevel);
+    expect(props.reactStyle).toStrictEqual({
+      visibility: {
+        styletype: 'constant',
+        stylevalue: {type: 'string', value: testProps.style.visibility},
+      },
+    });
+  });
+  // debug();
```
Comment (ferdicus) [inline __tests__/components/HeatmapLayer.test.js]: feel free to remove this line :)

File: __tests__/components/HeatmapLayer.test.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import {render} from 'react-native-testing-library';
+
+import HeatmapLayer from '../../javascript/components/HeatmapLayer';
+
+export const NATIVE_MODULE_NAME = 'RCTMGLHeatmapLayer';
```
Comment (mfazekas) [inline __tests__/components/HeatmapLayer.test.js]: this looks to be unused

Comment (viddity) [reply __tests__/components/HeatmapLayer.test.js]: Thanks! Is removed.
Sorry for the sloppiness. Got too used to TS warnings it seems.

File: __tests__/components/HeatmapLayer.test.js (hunk 1 above)
Comment (viddity) [reply __tests__/components/HeatmapLayer.test.js]: Njag.. Sloppy Sloppy... is removed.

File: __tests__/components/HeatmapLayer.test.js (hunk 2 above)
Comment (ferdicus) [reply __tests__/components/HeatmapLayer.test.js]: `eslint` does warn if the `export` is removed, however with the `export` you never can be sure if the var is used or not 🤷‍♂

**Pull request merged**